### PR TITLE
Please remove

### PR DESCRIPTION
**Pull request by mikebdp2**
Is replaced by a GPG-signed https://github.com/3mdeb/news-and-ideas/pull/397

### CODE REVIEW

**Comment by mikebdp2** [comment]
Just added the OSF_vPub.png cover.

File: blog/content/post/2021-02-23-OSF_vPub_01.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Together with our prominent 3mdeb hosts - Piotr Król and Michał Żygowski - it
+was an honour to be visited by such rockstars as Thierry Laurion from [Insurgo][1],
+Tom Hiller aka Thrilleratplay, Daniel Maslowski aka CyReVolt, Piotr Kubaj,
+Andrew Cooper, Felix Singer, David Hendricks, and many others whom I forgot to
+mention (sorry!) - or who preferred to stay anonymous and participate via chat.
+
+We've also had some surprise guests: despite being the relative newcomers to our
+cosy community, they have managed to ignite quite an interesting discussion!
+Alexey Vazhnov described how tricky could it be to get merged the coreboot docs,
+and we've thought about how to improve this workflow & make this process easier.
```
Comment (pietrushnic) [inline blog/content/post/2021-02-23-OSF_vPub_01.md:36]: @miczyg1 did we managed to do something with that? @mikebdp2 can we link discussion from reddit?

File: blog/content/post/2021-02-23-OSF_vPub_01.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+we've tried to figure out how to promote the coreboot for its official support
+by the PC manufacturers - at least for those with new AMD-based motherboards!
+And now we know, how we can - even if not "bring a coreboot to every house" -
+but to significantly spread its' popularity by wisely doing our joint activism.
+
+Also, we've talked a lot about the open hardware: for example, two interesting
+boards from Crowdsupply - [ULX3S][3] and [Qomu][4] - have a small FPGA for which
+an opensource toolchain is available. Also, we've explored the firmware freedom
+in a "world without x86", [Talos II][5] workstation with a CPU(s) of POWER9 arch,
+could POWER10 become as libre as POWER9 regarding the firmware, and much more!
```
Comment (pietrushnic) [inline blog/content/post/2021-02-23-OSF_vPub_01.md:48]: @mikebdp2 D-DIMM topic was big part of that, so it would be great to have link here.

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: blog/content/post/2021-02-23-OSF_vPub_01.md (hunk 2 above)
Comment (mikebdp2) [reply blog/content/post/2021-02-23-OSF_vPub_01.md]: @pietrushnic Done

File: blog/content/post/2021-02-23-OSF_vPub_01.md (hunk 1 above)
Comment (mikebdp2) [reply blog/content/post/2021-02-23-OSF_vPub_01.md:36]: @pietrushnic Done

**Review by pietrushnic — APPROVED** [review]

**Comment by pietrushnic** [comment]
@mikebdp2 I'm waiting for external feedback but it looks good for me now.

**Review by miczyg1 — APPROVED** [review]

**Pull request closed without merging**